app/Post_Type/Hero.php

```diff
@@ -31,7 +31,9 @@ public function init() {
 
 		$this->has_single    = false;
 		$this->is_searchable = false;
-		$this->redirect      = home_url();
+		// redirect is commented - to hide view link on the admin panel
+		// do this if has_single == false
+		//$this->redirect      = home_url();
```

Comment:
please investigate first why this redirect is set here!

I dont' think you were right commenting it.

Reply:
A redirect is needed so that you can not make the transition to a post of a closed content type